manifests/opsmanager.pp

```diff
@@ -0,0 +1,52 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::opsmanager::params::user,
+  String[1] $group                               = $mongodb::opsmanager::params::group,
+  Enum['running', 'stopped'] $ensure             = $mongodb::opsmanager::params::ensure,
+  String[1] $package_name                        = $mongodb::opsmanager::params::package_name,
+  String[1] $package_ensure                      = $mongodb::opsmanager::params::package_ensure,
+  Boolean $service_enable                        = $mongodb::opsmanager::params::service_enable,
+  Boolean $service_manage                        = $mongodb::opsmanager::params::service_manage,
+  String[1] $service_name                        = $mongodb::opsmanager::params::service_name,
+  Stdlib::Httpurl $download_url                  = $mongodb::opsmanager::params::download_url,
+  String[1] $mongo_uri                           = $mongodb::opsmanager::params::mongo_uri,
+  Stdlib::Httpurl $opsmanager_url                = $mongodb::opsmanager::params::opsmanager_url,
+  String[1] $client_certificate_mode             = 'None',
+  String[1] $from_email_addr                     = 'from@yourdomain.com',
```

Comment:
Could you change these defaults to at least `@example.com`? If users are expected to set these, we can also leave them empty.

Reply:
Fixed